platform/android/detect.py

```diff
@@ -200,27 +201,27 @@ def configure(env: "SConsEnvironment"):
         # The NDK adds this if targeting API < 24, so we can drop it when Godot targets it at least
         env.Append(CCFLAGS=["-mstackrealign"])
         if has_swappy:
-            env.Append(LIBPATH=["../../thirdparty/swappy-frame-pacing/x86"])
+            env.Append(LIBPATH=["#thirdparty/swappy-frame-pacing/x86"])
     elif env["arch"] == "x86_64":
         if has_swappy:
-            env.Append(LIBPATH=["../../thirdparty/swappy-frame-pacing/x86_64"])
+            env.Append(LIBPATH=["#thirdparty/swappy-frame-pacing/x86_64"])
     elif env["arch"] == "arm32":
-        env.Append(CCFLAGS="-march=armv7-a -mfloat-abi=softfp".split())
+        env.Append(CCFLAGS=["-march=armv7-a", "-mfloat-abi=softfp"])
         env.Append(CPPDEFINES=["__ARM_ARCH_7__", "__ARM_ARCH_7A__"])
         env.Append(CPPDEFINES=["__ARM_NEON__"])
         if has_swappy:
-            env.Append(LIBPATH=["../../thirdparty/swappy-frame-pacing/armeabi-v7a"])
+            env.Append(LIBPATH=["#thirdparty/swappy-frame-pacing/armeabi-v7a"])
     elif env["arch"] == "arm64":
         env.Append(CCFLAGS=["-mfix-cortex-a53-835769"])
         env.Append(CPPDEFINES=["__ARM_ARCH_8A__"])
         if has_swappy:
-            env.Append(LIBPATH=["../../thirdparty/swappy-frame-pacing/arm64-v8a"])
+            env.Append(LIBPATH=["#thirdparty/swappy-frame-pacing/arm64-v8a"])
 
     env.Append(CCFLAGS=["-ffp-contract=off"])
 
     # Link flags
 
-    env.Append(LINKFLAGS="-Wl,--gc-sections -Wl,--no-undefined -Wl,-z,now".split())
+    env.Append(LINKFLAGS=["-Wl,--gc-sections", "-Wl,--no-undefined", "-Wl,-z,now"])
     env.Append(LINKFLAGS="-Wl,-soname,libgodot_android.so")
```

Comment:
While you're refactoring these:
```suggestion
    env.Append(LINKFLAGS=["-Wl,-soname,libgodot_android.so"])
```